modules/test/playwright/pages/users-admin-web/AccountSettingsPage.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+// @ts-ignore
+
+import {Locator, Page} from '@playwright/test';
+
+export class AccountSettingsPage {
+    readonly accountSettingsMenuItem: Locator;
+    readonly page: Page;
+    readonly userPersonalMenuButton: Locator;
+    readonly rolesMenuItem: Locator;
+    readonly saveButton: Locator;
+
+    constructor(page: Page) {
+        this.accountSettingsMenuItem = page.getByRole('menuitem', {
+                name: 'Account Settings'
+            });
+        this.userPersonalMenuButton = page.getByTestId('userPersonalMenu');
+        this.rolesMenuItem = page.getByRole('link', {
+                name: 'Roles'
+            });
+        this.saveButton = page.getByRole('button', {
+                name: 'Save'
+            });
+        this.page = page;
```

Comment:
You need to sort all of these constructors by alphabetical order. cc/ @smottal

Reply:
Pretty sure this is why it is failing SF @wanderlast

Reply:
I was finally able to get the prettifier to touch the files locally. It appears to have actually been issues with tabs/spacing...?  I'll have to look closer at the requirements since I don't know why it tripped now but not before. Finally passed SF now though, so I'll re-run the playwright suite as well. @brianikim